plugin/rasterizing/rasterizing/src/main/java/org/geomajas/plugin/rasterizing/tms/TmsService.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * This is part of Geomajas, a GIS framework, http://www.geomajas.org/.
+ *
+ * Copyright 2008-2014 Geosparc nv, http://www.geosparc.com/, Belgium.
+ *
+ * The program is available in open source according to the GNU Affero
+ * General Public License. All contributions in this program are covered
+ * by the Geomajas Contributors License Agreement. For full licensing
+ * details, see LICENSE.txt in the project root.
+ */
+package org.geomajas.plugin.rasterizing.tms;
+
+import java.io.Writer;
+
+import org.geomajas.global.GeomajasException;
+
+/**
+ * The TMS service is responsible for publishing the TMS metadata (vector layers only!). See
+ * {@link org.geomajas.plugin.rasterizing.mvc.TmsController} for the actual rendering of the tiles.
+ * 
+ * @author Jan De Moerloose
+ *
+ */
+public interface TmsService {
```

Comment:
I consider it a TMS service (or metadata service) but it looks indeed like the focus is on the writing now. To be reevaluated when we turn it into API ?